app/ts/components/requirements.tsx

```diff
@@ -205,9 +233,28 @@ export const Create = ( { contentHashInput, resolutionAddressInput, loadingInfos
 		return first.domainInfo.expiry.toISOString() !== last.domainInfo.expiry.toISOString()
 	})
 
-	return <>
-		<div style = 'padding-top: 10px;'>
-			{ !immutable.value ? <div style = 'padding: 10px;'>
+	if (checkBoxes.deepValue === undefined) return <></>
+	const subDomain = checkBoxes.deepValue[checkBoxes.deepValue.length - 1]?.domainInfo.subDomain
+	if (subDomain === undefined) throw new Error('missing subdomain')
+
+	return <div style = 'padding-top: 10px;'>
```

Comment:
ternaries multiple nested conditions tend to be a little more readable with `if` statements

Reply:
My preference for giant complicated JSX like this is to use local components with `useState(() => () => <>...</>)`